tests/functional/tokenstorage_v2/test_json_token_storage.py

```diff
@@ -11,25 +11,25 @@
 
 
 @pytest.fixture
-def filename(tempdir):
-    return os.path.join(tempdir, "mydata.json")
+def json_file(tmp_path):
+    return tmp_path / "mydata.json"
```

Comment:
This could migrate to a higher-level `conftest.py` fixture since it's duplicated.

Reply:
Maybe, but I'm also not bothered by the duplication, since it's pretty small/minor. For now I'm leaving it as-is.